cli-config/src/config.rs

```diff
@@ -17,9 +22,47 @@ lazy_static! {
     ///
     /// [lazy_static]: https://docs.rs/lazy_static
     pub static ref CONFIG_FILE: Option<String> = {
-        dirs_next::home_dir().map(|mut path| {
+        let config_override_path = dirs_next::home_dir().map(|mut path| {
```

Comment:
instead of a second statically defined config filepath, can we implement this as an optional config field and load it when declared?

Reply:
I'll look into that